.travis.yml

```diff
@@ -44,7 +44,7 @@ script:
   - cd $TRAVIS_BUILD_DIR
   # Static type checking for Python, for now only as warning and when mypy is
   # available (which it isn't in PyPy).
-  - if [ -x "$(command -v mypy)" ]; then mypy django/ || true; fi
+  - if [ -x "$(command -v mypy)" ]; then mypy django/ ; fi
```

Comment:
With the `|| true` gone, it seems like the ", for now only as warning and" in the line above can be removed now?